website/docs/static/luigi/coreStyles.bundle.js

```diff
@@ -0,0 +1,61 @@
+/*!
+ *
+ *       Don't be afraid!
+ *       This file was generated automatically and you should not modify it.
```

Comment:
same with **website/docs/static/luigi-client/** , this folder:  **website/docs/static/luigi/** contains auto-generated files. I don't think any of them should be included in this PR at all. I can only presume they are created at runtime only. Please remove accordingly and/or add them as needed in the respective **.gitignore**